cmd/fireaptos/cli/firehose.go

```diff
@@ -97,7 +97,7 @@ func init() {
 
 				sss := substreamsService.New(
 					stateStore,
-					"sf.aptos.type.v1.Transaction",
+					"sf.aptos.type.v1.Block",
```

Comment:
I think we do indeed. The actual type will read `aptos.extractor.v1.Block`, I'm wondering how it's the feeling of this. I'll update to fit with the new name for now for consistency.